_posts/2020-12-01-kafka-cloud-events.adoc

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title: Sending and Receiving Cloud Events with Kafka
+date: 2020-11-24
+tags: reactive kafka
+synopsis: How to send and receive Cloud Events with Quarkus, Kafka and Reactive Messaging
+author: cescoffier
+---
+
+https://cloudevents.io/[Cloud Events] is a specification for describing events.
+It aims to ease interoperability.
+With the rise of event-driven architecture, it's not surprising to see Cloud Events gaining popularity.
+
+This post explains how you can generate and consume Cloud Events using Quarkus, Kafka, and Reactive Messaging.
+
+== What is Cloud Event?
+
+First, let's look at the why.
+Events are everywhere.
+Many modern systems are using events one way or another.
+Events can be used to implement event sourcing, communicate facts, trigger out-of-band processing, or send notifications.
+Events become an essential piece of any system.
+
+Yet event publishers tend to describe events differently.
+I don't mean the content is different, but the envelope and the event's format are heterogeneous, even if these events transit on the same event mesh.
+Some applications choose JSON and encode everything in the event's payload; some other systems prefer binary formats, such as Avro or Protobuf, and use the protocol capability, such as headers or properties,  to transport metadata about the wrapped payload.
+While event-driven architecture claims to ease the integration with external systems, this disparity is doing the opposite.
+It's not rare to need a specific event translator with the only purpose to adapt events from one format to another.
+
+So, what's Cloud Event?
+Cloud Event proposes a common way to describe events.
+The goal is, obviously, interoperability and facilitating the integration burden.
```

Comment:
Maybe "facilitating" should be "easing", as you're saying the integration burden should be easier and not harder